[docs] update Register a tax certificate (#20585)
docker/docs: content/billing/tax-certificate.md

## Register a tax certificate

1. [Submit a Docker Support ticket](https://hub.docker.com/support/contact?topic=Billing&subtopic=Tax%20information) to initiate the process to register a tax certificate.
2. Enter the required information.
3. In the **Additional Information** field, list the Docker ID/namespace(s) of the accounts that you want to apply the tax exemption certificate to.  

   > **Tip**
   >
   > You can list multiple namespaces that share the same tax exemption certificate, if applicable.
   { .tip }

> 4. Select **Browse Files** to upload the tax certificate from your system or simply drag and drop the certificate in the given space.

Reviewer: ```suggestion
4. Add the tax certificate from your system by dragging and dropping them onto the file area, or select the **Browse Files** button to open a file dialog.
```

Author: Done. Thank you.

5. Select **Submit**.

> Docker's Support team will reach out to you once a ticket is open and if Support team needs any additional information. You'll receive a confirmation from Docker once your tax exemption status has processed and is applied to your account.

Reviewer: ```suggestion
Docker's support team will reach out to you if any additional information is required. You'll receive an e-mail confirmation from Docker once your tax exemption status is applied to your account.
```

Author: Done. Thank you.

## Register a tax certificate

1. [Submit a Docker Support ticket](https://hub.docker.com/support/contact) to initiate the process to register a tax certificate.
2. Enter the required information.
3. Select **Billing** from the **Topic** drop-down.

> 4. Select **Tax Information** from the **Subtopic** drop-down.

Reviewer: Can we add query parameters to the contact url to pre-select these fields?

Reviewer: Yeah something like that

Author: Done. Thank you.
